### DISCUSSION TITLE
web3 modal v3 account view(overlay screen) causing ui to flicker

### DISCUSSION THREAD

**Discussion by ethdev279** [WalletConnect/walletconnect-monorepo#3722] [Developer Support]
web3 modal v3 account view(overlay screen) causing ui to flicker when there is a constant update to ui. I think this was due to improper zindex?

https://github.com/WalletConnect/walletconnect-monorepo/assets/45661693/4764425a-3ac2-4072-9701-109cf2d0aa12

**Comment by glitch-txs** [comment]
Hey that's weird, but you can set a z-index in the theming options

https://docs.walletconnect.com/web3modal/react/theming

**Reply by ethdev279** [reply]
Tried. That didn't work.

**Reply by glitch-txs** [reply]
Do you have a public repo or MRE that we can run and test?

**Reply by ethdev279** [reply]
Here it is. Try hovering on the edges of modal or try switching network from account view and then move cursor away. you'll notice the issue. https://stackblitz.com/edit/stackblitz-starters-wrqnsn?file=app%2Fcomponents%2FWeb3Provider.js

**Reply by glitch-txs** [reply]
works fine for me
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/66949816/bca34901-e3d6-4971-adf3-467fe6011624)

**Reply by glitch-txs** [reply]
have you tried on another browser? or reinstalling it? other computer?

**Comment by ethdev279** [comment]
Looks like there was something wrong with my chrome(or an extnsion in it causing the issue). Thanks for the help btw!
